packages/web-components/src/components.d.ts

```diff
@@ -4286,6 +4290,10 @@ declare namespace LocalJSX {
         "inline"?: boolean;
         "onIcNavigationMenuClosed"?: (event: IcTopNavigationCustomEvent<void>) => void;
         "onIcNavigationMenuOpened"?: (event: IcTopNavigationCustomEvent<void>) => void;
+        /**
+          * The short title of the app to be displayed at small screen sizes in place of the regular app title.
```

Comment:
```suggestion
          * The short title of the app to be displayed at small screen sizes in place of the default app title.
```

I feel as though regular is not the right word, but then I'm not sure if default is the right word either. Might be worth getting a second opinion on this.